qa/t/test_user_preferences.t

```diff
@@ -249,7 +249,6 @@ $sel->type_ok("comment",
 $sel->click_ok("commit");
 $sel->wait_for_page_to_load(WAIT_TIME);
 $sel->is_text_present_ok("Changes submitted for bug $bug1_id");
-$sel->is_text_present_ok("The next bug in your list is bug $bug2_id");
```

Comment:
I think this test has been (for a while now) made obsolete by the 'bug/navigate' template partial (which shows the current/next/previous bug in the bug list your browsing). This only appears on the old bug UI as well and it is of no value IMO (it appears on the same page of the next bug, so you're already on the next bug page when you see it...).